### PR TITLE
Make readability edits

### PR DESCRIPTION
**Pull request by alexcameron89**
As I was reading through, I made a few edits that I thought would help it flow better.

### CODE REVIEW

**Comment by alexcameron89** [comment]
Also, is there a trick to getting the text to realign? I know that some of my lines are short and long.

**Comment by steveklabnik** [comment]
> Also, is there a trick to getting the text to realign? I know that some of my lines are short and long.

What editor do you use?

File: src/editions/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-After a while, all of those tiny changes add up. It can be hard to look back
+After a while, all of those tiny changes add up. But, from release to release, it can be hard to look back
 and say *"Wow, between Rust 1.10 and Rust 1.20, Rust has changed a lot!"*
 Furthermore, since everything must be backwards compatible, we can't make certain
-tiny changes, like adding keywords. Finally, various parts of the project,
-since they update at different paces, may feel like they "lag" in various ways.
-If a new language feature lands in Rust 1.15, and a new chapter of the book
-on it lands in 1.16, and the IDE integration lands in 1.17, this can feel
-very disjointed. But if you were to compare Rust 1.15 to Rust 1.17, you'd see
+tiny changes, like adding keywords. Finally, you may sense lag through the various parts of the project,
```
[the remark this answers is not in the document]
Comment (alexcameron89) [reply src/editions/index.md]: Good point, changed!

**Comment by alexcameron89** [comment]
Hey Steve! 👋 I'm using Vim. Is there a secret plugin/command that I don't know yet?

**Comment by Centril** [comment]
LGTM 👍

**Pull request merged**